src/Security/PasswordValidator.php

```diff
@@ -89,8 +88,7 @@ public function minLength($minLength)
     }
 
     /**
-     * @deprecated 4.5.0
-     * Check the character strength of the password.
+     * @deprecated 4.5.0 Check the character strength of the password.
```

Comment:
```suggestion
     * Check the character strength of the password.
     * @deprecated 4.5.0 Use setMinTestScore() and setTestNames() instead.
```
Update the notice message as well.

Reply:
Updated